packages/gestalt/src/Popover/Contents.tsx

```diff
@@ -54,12 +66,18 @@ export default function Contents({
   overflow,
 }: Props) {
   const caretRef = useRef<HTMLElement | null>(null);
-  const idealPlacement = idealDirection ? DIRECTIONS_MAP[idealDirection] : 'top';
+
+  let idealPlacement: 'top' | 'right' | 'bottom' | 'left' = 'top';
```

Comment:
Maybe this is more accurate: 

```suggestion
  let idealPlacement: Side = 'top';
```

Reply:
DONE